toc.md

```diff
@@ -62,6 +62,8 @@
     - [Creating a Deployment](./3-tutorial.md#creating-a-deployment)
     - [Making Your Deployment Accessible](./3-tutorial.md#making-your-deployment-accessible)
         - [Exposing Ports](./3-tutorial.md#exposing-ports)
+            - [Communication in a Pod](#communication-in-a-pod)
```

Comment:
Going to be picky... `in a Pod` should actually be `within a Pod`

Reply:
I prefer your wording, one sec

Reply:
fixed